components/AdvocateCard.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <article>
```

Comment:
Add a class (`advocate-card`, for instance) to allow external CSS to adjust the visualization of the component.